### PR TITLE
Fixed issue #19056: save button in create dummy participants page actually saves and closes

### PR DESCRIPTION
**Pull request by gabrieljenik**

### CODE REVIEW

File: application/controllers/admin/Tokens.php [hunk 1]
```diff
@@ -1081,7 +1081,8 @@ public function addDummies(int $iSurveyId, string $subaction = '')
                 [
                     'showSaveAndCloseButton' => true,
```
Comment (Shnoulle) [inline application/controllers/admin/Tokens.php:1082]: No way to renane "Save and close" to "Create" ?

**Review by Shnoulle — APPROVED** [review]
If no way : need to create an UX issue. It muts be “Create participants“

File: application/controllers/admin/Tokens.php (hunk 1 above)
Comment (gabrieljenik) [reply application/controllers/admin/Tokens.php:1082]: Text is not customizable. But could it be... let's leave it for another ticket

Comment (Shnoulle) [reply application/controllers/admin/Tokens.php:1082]: Yep

> If no way : need to create an UX issue. It muts be “Create participants“

**Pull request merged**